ajax/libs/simplebar/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "simplebar",
+  "filename": "simplebar.js",
+  "description": "Scrollbars, simpler.",
+  "homepage": "https://grsmto.github.io/simplebar/",
+  "keywords": [
+    "simplebar",
+    "scrollbar"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/grsmto/simplebar.git"
```

Comment:
Let's use `https`.
Using https is better because it's more security for browsing. to prevent the possibility the users been attacked during the redirection.
Thank you.